src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    static final String PATTERN = "dd.MM.yyyy";
```

Comment:
```suggestion
    private static final String PATTERN = "dd.MM.yyyy";
```

не забывай про модификаторы доступа

Reply:
Да, спасибо, понял, подправлю